src/main/java/com/google/api/codegen/discovery/transformer/nodejs/NodeJSSampleTypeNameConverter.java

```diff
@@ -26,6 +26,7 @@
   /** A map from primitive types in proto to NodeJS counterparts. */
   private static final ImmutableMap<Field.Kind, String> PRIMITIVE_TYPE_MAP =
```

Comment:
Sorry, I forgot we were using these type names in the dynamic languages to construct meaningful identifier names for, e.g., iteration variables. Has this functionality been preserved? (It may not be well exercised in the baselines.)